<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWidget</class>
 <widget class="QWidget" name="SettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>94</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="backButton">
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="go-previous"/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QRadioButton" name="recorderLocalButton">
        <property name="text">
         <string>Local</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QRadioButton" name="transcriberLocalWhisperButton">
        <property name="text">
         <string>Local</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QRadioButton" name="recorderRemoteButton">
        <property name="text">
         <string>Remote</string>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QRadioButton" name="trancriberOpenAIApiWhisperButton">
        <property name="text">
         <string>OpenAI Api</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QRadioButton" name="transcriberRemoteWhisperButton">
        <property name="text">
         <string>Remote</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
